packages/ncids-js/src/components/nci-header/__tests__/search/search.test.ts

```diff
@@ -56,40 +56,6 @@ describe('NCI Search', () => {
 		header.unregister();
 	});
 
-	it('check that event listeners are added and removed', async () => {
```

Comment:
Removed because this is a copy of 

```bash
 NCI Extended Header
    ✓ should add event listeners 
    ✓ should add event listeners
```

https://github.com/NCIOCPL/ncids/blob/develop/packages/ncids-js/src/components/nci-header/__tests__/nci-header.test.ts#L77